docs/rest_api_2.rst

```diff
@@ -0,0 +1,4156 @@
+REST API
+========
+
+This section will go over the specifies of the REST APIs supported by F5 ACI ServiceCenter application.
+
+The REST calls can be made to the following APIC endpoint
+
+.. code-block:: json
+
+   https://<APIC-URL>/appcenter/F5Networks/F5ACIServiceCenter/<REST-API>
+
+   Example:
+
+   https://10.107.0.24/appcenter/F5Networks/F5ACIServiceCenter/getbigiplist.json
+
+`Download the API POSTMAN collection <https://github.com/F5Networks/f5-aci-servicecenter/tree/master/api_collection>`_.
+
+Request Headers
+---------------
+
+For all the F5 ACI ServiceCenter API calls following request header parameters are required:
```

Comment:
...API calls, the following request header...

Reply:
Resolved in docs/rest_api.rst (docs/rest_api_ep.rst  is an outdated filename)